n3fit/src/n3fit/tests/test_preprocessing.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+from n3fit.layers import Preprocessing
+
+
+def test_preprocessing():
+    """Regression test"""
+    # taken from basic runcard
+    flav_info = [
+        {'fl': 'sng', 'smallx': [1.05, 1.19], 'largex': [1.47, 2.7], 'trainable': False},
+        {'fl': 'g', 'smallx': [0.94, 1.25], 'largex': [0.11, 5.87], 'trainable': False},
+        {'fl': 'v', 'smallx': [0.54, 0.75], 'largex': [1.15, 2.76], 'trainable': False},
+        {'fl': 'v3', 'smallx': [0.21, 0.57], 'largex': [1.35, 3.08]},
+        {'fl': 'v8', 'smallx': [0.52, 0.76], 'largex': [0.77, 3.56]},
+        {'fl': 't3', 'smallx': [-0.37, 1.52], 'largex': [1.74, 3.39]},
+        {'fl': 't8', 'smallx': [0.56, 1.29], 'largex': [1.45, 3.03]},
+        {'fl': 'cp', 'smallx': [0.12, 1.19], 'largex': [1.83, 6.7]},
+    ]
+    prepro = Preprocessing(flav_info=flav_info, seed=0)
```

Comment:
```suggestion
    prepro = Preprocessing(flav_info=flav_info, seed=1)
```

you never know when someone wrote `if seed:` in their code :P